### PR TITLE
Update music options FE logic & light refactor

### PR DESCRIPTION
**Pull request by paladique**
#19 did not fix the the intended bug, but the front end logic will still prevent users from trying to change ac music if they opt for K.K. Slider to play at all times. Includes some light refactoring.

### CODE REVIEW

File: options.html [hunk 1]
```diff
@@ -101,16 +101,19 @@ <h4>K.K. Slider - Play Times</h4>
...
-							<input id="kk-version-live" name="kk-version" type="radio"><label for="kk-version-live"><span><span></span></span>Live</label>
-						</div>
-						<div class="radio">
-							<input id="kk-version-aircheck" name="kk-version" type="radio"><label for="kk-version-aircheck"><span><span></span></span>Aircheck</label>
-						</div>
-						<div class="radio">
-							<input id="kk-version-both" name="kk-version" type="radio"><label for="kk-version-both"><span><span></span></span>Both</label>
-						</div>
+						
+						<section id="kk-version-selection">
```
Comment (PikaDude) [inline options.html]: image
This creates a peculiar indentation.

**Review by PikaDude — CHANGES_REQUESTED** [review]

**Comment by PikaDude** [comment]
image
Additionally, we might as well disable the weather options since they are also irrelevant if only K.K. slider is playing.

File: options.html (hunk 1 above)
Comment (paladique) [reply options.html]: Ah good catch, should be fixed.

**Comment by paladique** [comment]
Very unsure how I rebased like that, but happy to pick my changes and add to #37 instead

**Comment by PikaDude** [comment]
Yeah, #37 changed a bit, so feel free to do that if you want.

**Comment by paladique** [comment]
Closing this! Will put in another branch and PR.

**Pull request closed without merging**